src/EventStore.Client/ConnectionStringParseException.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+
+namespace EventStore.Client {
+	public class ConnectionStringParseException : Exception {
```

Comment:
Can we put each of these into their own file? It would make finding it on github easier.

Reply:
yup, it has been fixed in this commit: da3a713c7f99e7a1297bfcd9452b37e7a2933be6

thanks